### DISCUSSION TITLE
Azure SDK function to list local users for storage account not giving output.

### DISCUSSION THREAD

**Discussion by Cyber-Squid1** [community/community#119633] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am trying to fetch user permissions on files stored on my azure storage account using the azure sdk provided in python. The code I used was from azure's online documentation (https://learn.microsoft.com/en-us/rest/api/storagerp/local-users/list?view=rest-storagerp-2023-01-01&tabs=Python) but the output I get is empty. Is there a specific permission that i need to provide my user to get the output of the function?

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @Cyber-Squid1 !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!
